app/HistoryExporter.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Exporter;
+
+use App\Repositories\HistoryRepository;
+use App\Repositories\SettingsRepository;
+
+class HistoryExporter
+{
+    protected $data;
+    protected $messages;
+    protected $lang;
+    protected $settingsRepository;
+    protected $jsonBaseRepository;
+
+    public function __construct()
+    {
+        $this->jsonBaseRepository = new HistoryRepository();
+        $this->data = $this->jsonBaseRepository->all();
+
+        $this->settingsRepository = new SettingsRepository();
+        $this->lang = $this->settingsRepository->getSetting('localization', 'locale');
+        $this->messages = $this->loadLocale($this->lang);
+    }
+
+    public function export($exportedData)
```

Comment:
let's get date of history to export as an function argument, if null - show all dates

```suggestion
    public function export($date = null)
```